src/Spinner/Spinner.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+
+import { Spinner } from './Spinner';
+import { SpinnerProps } from './types';
+import { ThemeProvider } from 'styled-components';
+import { theme } from '../shared/theme';
+
+export default {
+  title: 'Components/Spinner',
+  component: Spinner,
+  parameters: {
+    componentSubtitle: 'Displays a Spinner',
+  },
+  args: {
+    size: 'M',
+    color: 'blue.main',
+    spinnerColor: 'neutral.300',
+  },
+} as ComponentMeta<typeof Spinner>;
+
+const Template: ComponentStory<typeof Spinner> = (args: SpinnerProps) => (
```

Comment:
Since there is only one story, maybe show the component directly instead of under `default`:

```
import React from 'react';
import { ComponentStory, ComponentMeta } from '@storybook/react';

import { Spinner as SpinnerComponent } from './Spinner';
import { SpinnerProps } from './types';
import { ThemeProvider } from 'styled-components';
import { theme } from '../shared/theme';

export default {
  title: 'Components/Spinner',
  component: SpinnerComponent,
  parameters: {
    componentSubtitle: 'Displays a Spinner',
  },
  args: {
    size: 'M',
    color: 'blue.main',
    spinnerColor: 'neutral.300',
  },
} as ComponentMeta<typeof SpinnerComponent>;

export const Spinner: ComponentStory<typeof SpinnerComponent> = (args: SpinnerProps) => (
  <ThemeProvider theme={theme}>
    <SpinnerComponent {...args} />
  </ThemeProvider>
);
```